frontend/src/utils/StatusUtils.ts

```diff
@@ -0,0 +1,16 @@
+// Change status mysql enum to title case
+function convertStatusTitleCase(status: string) {
+  const statusEnum = status.substring(30);
+  switch (statusEnum) {
+    case "DEFAULT":
+      return "Pending";
+    case "APPROVED":
+      return "Approved";
+    case "ACTION_REQUIRED":
+      return "Action Required";
+    default:
+      return statusEnum[0] + statusEnum.substring(1).toLowerCase();
```

Comment:
Not that there's an explicit case where I think this happens, but this won't help us with anything with an additional underscore. For completion sake, think it makes more sense to split + join along underscores? 

eg. `THIS_WOULD_BECOME` -> "This_would_become"

Reply:
now changed to 
`THIS_WOULD_BECOME `-> `This would become`

Reply:
Would we rather `THIS_WOULD_BECOME` -> `This Would Become`? That's what action required does